application/scss/components/_copy-button.scss

```diff
@@ -0,0 +1,25 @@
+.app-link--copy {
+  &:link,
+  &:active,
+  &:visited,
+  &:hover {
+    @include govuk-font(16);
+    position: absolute;
```

Comment:
This relies on the implicit `position` from `_tabs.scss`, is that safe/reliable enough?